README.md

```diff
@@ -23,9 +23,9 @@ If you want to try out _Fredy_, you can access the demo version [here](https://f
 - Make sure to use Node.js 20 or above
 - Run the following commands:
 ```ssh
-yarn (or npm install)
```

Comment:
I would not advice for or offer support for another package manager here, but default to Yarn. 

Also `npm install` will create a `package-lock.json` file which is not in the `.gitignore`.